model/project_vars.go

```diff
@@ -61,6 +64,28 @@ type ProjectVars struct {
 	AdminOnlyVars map[string]bool `bson:"admin_only_vars" json:"admin_only_vars"`
 }
 
+type ParameterMappings []ParameterMapping
+
+// NameMap returns a map from each name to the full parameter mapping
+// information.
+func (pm ParameterMappings) NameMap() map[string]ParameterMapping {
```

Comment:
nit: can unexport

Reply:
Hmm I was thinking this could be useful in the future since we'll need this kind of DB field <-> parameter mapping for other secret fields in the future. It'll also help with testing in other packages in upcoming PRs, like the REST routes for project vars.